cli/common_test.go

```diff
@@ -42,6 +44,15 @@ func Test_Common_createRequest(t *testing.T) {
 			},
 		},
 
+		// This test ensures that trying to load unit files with invalid input
```

Comment:
I like the test - how about another one with a name as the Input but an empty file system?